packages/node/CHANGELOG.md

```diff
@@ -931,7 +938,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Changed
 - bump @polkadot/api to 3.1.1
 
-[Unreleased]: https://github.com/subquery/subql/compare/node/2.10.0...HEAD
+[Unreleased]: https://github.com/subquery/subql/compare/node/2.11.0...HEAD
+[2.11.0]: https://github.com/subquery/subql/compare/node/2.10.0...node/2.11.0
```

Comment:
I think this can be a patch version, all the changes are bug fixes or removing redundant code